script.js

```diff
@@ -2,6 +2,13 @@
 let restaurantsApp = {}
 //Store API key
 restaurantsApp.apiKey = '2-RCWO0-I-9m7PMN2zt0fcZ45itXKXWRfnBQtimCYUjh2skNC9-_CAF_SJdwlTkeymvzhlzSyQFDz0kih-S3Cjz1JIxklzgXrnO-YySwD4ThKeBFlskagdf0JeeVXnYx';
+// Create empty arrays to store businesses information
+restaurantsApp.businessID=[]
+restaurantsApp.businessName=[]
+restaurantsApp.businessAddress=[]
+restaurantsApp.businessRating=[]
+restaurantsApp.businessPrice=[]
+restaurantsApp.businessReviews=[]
```

Comment:
why did you choose arrays and remove the for loop? the results should be placed immediately onto the page and do not need to be stored in an array unless you are planning to use arrays for another feature which we haven't discussed yet.

Reply:
that is because in for loop each time the for loop runs the variable is over written and the info from previous business is lost. this way we have access to to info from all businesses to be displayed.

Reply:
i changed it slightly in my PR because it wasn't finished [latest PR](#4). if you look at my pr you'll see that every time we loop over the result we immediately put it on the page. this means that we won't need any arrays to keep track of the business info.